src/components/pages/errorPage/errorPage.ts

```diff
@@ -0,0 +1,26 @@
+import { Container } from 'constants/htmlConstants';
+import './errorPage.scss';
+
+class ErrorPage {
+  constructor(private container: HTMLElement = Container) {
+    this.renderPage();
+  }
+
+  public renderPage(): void {
+    const errorPageContainer: HTMLElement = document.createElement('div');
+    errorPageContainer.classList.add('error');
+    errorPageContainer.innerHTML = `
+      <div class="error__descr">
+        <p>Error 404!</p>
+        <span>Something went wrong...</span>
+      </div>
+      `;
+    this.container.append(errorPageContainer);
+  }
+
+  public removePage(): void {
+    this.container.innerHTML = '';
+  }
+}
```

Comment:
Pretty weird class. It contains `render` method (it's public) but at the same time it calls the method inside a constructor. Just call the method from the instance, your constructor should not incapsulate your public methods